android-npm/build.gradle

```diff
@@ -60,16 +82,16 @@ rootProject.getSubprojects().forEach({project ->
                 project.getProperties().android.packagingOptions.pickFirst("lib/**/libfbjni.so")
 
                 replaceSoTaskDebug.dependsOn(
-                    project.getTasks().getByPath("${appName}:mergeDebugNativeLibs"),
-                    project.getTasks().getByPath("${appName}:stripDebugDebugSymbols")
+                    project.getTasks().getByPath("${appName}:merge${flavorString}DebugNativeLibs"),
```

Comment:
What about build variant does it also has an impact on the task name?

Reply:
I didn't understand a build variant term properly. It turned out that it's just a combination of build type and a flavor.